<ui version="4.0" >
 <author>José Manuel Díez Botella</author>
 <class>copiaseg</class>
 <widget class="QDialog" name="copiaseg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Copia de seguidad</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>FICHERO A GENERAR:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>nombre fichero</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
     <property name="textVisible" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton" >
       <property name="text" >
        <string>&amp;Aceptar</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/graf/1rightarrow.png</normaloff>:/graf/1rightarrow.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton" >
       <property name="text" >
        <string>&amp;Cancelar</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/images/librok-7.png</normaloff>:/images/librok-7.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="libroK.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>copiaseg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>356</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
